modules/team_planner/spec/features/team_planner_remove_event_spec.rb

```diff
@@ -33,7 +33,7 @@
 RSpec.describe 'Team planner remove event', js: true, with_ee: %i[team_planner_view] do
   include_context 'with team planner full access'
 
-  let!(:viewer_role) { create(:existing_role, permissions: [:view_work_packages]) }
+  let!(:viewer_role) { create(:project_role, permissions: [:view_work_packages]) }
```

Comment:
Same as above:
```suggestion
  let!(:viewer_role) { create(:existing_project_role, permissions: [:view_work_packages]) }
```

Reply:
I think the :existing_project_role factory capabilities are not needed in the specs I touched. The only reason for using it is IMO when wanting to add permissions to an already existing role which I don't see in those specs.